sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -175,6 +175,9 @@ object FunctionRegistry {
     expression[NullIf]("nullif"),
     expression[Nvl]("nvl"),
     expression[Nvl2]("nvl2"),
+    expression[OuterExplode]("outer_explode"),
+    expression[OuterInline]("outer_inline"),
```

Comment:
Why add this one to the function registry? It is not exposed in `functions.scala`.